tests/accessor/accessor_common.h

```diff
@@ -139,30 +139,64 @@ inline auto add_vectors_to_type_pack(StrNameType type_name) {
                                   "vec<" + type_name + ", 16>");
 }
 
+template <accessor_type AccType>
+struct tag_factory {
```

Comment:
Would you mind to move the `accessor_type` enum declaration before it's used? There would be compilation failure on linux otherwise.

Reply:
Fixed